static/app/components/nav/utils.tsx

```diff
@@ -0,0 +1,165 @@
+import {type HTMLProps, useMemo} from 'react';
+import type {LocationDescriptor} from 'history';
+
+import {isItemActive} from 'sentry/components/sidebar/sidebarItem';
+import {SIDEBAR_NAVIGATION_SOURCE} from 'sentry/components/sidebar/utils';
+import type {FeatureDisabledHooks, WithoutPrefix} from 'sentry/types/hooks';
+import normalizeUrl from 'sentry/utils/url/normalizeUrl';
+import type {useLocation} from 'sentry/utils/useLocation';
+
+export const NAV_DIVIDER = Symbol('divider');
+
+interface FeatureCheck {
+  features: string | string[];
+  hook?: WithoutPrefix<keyof FeatureDisabledHooks>;
+}
+
+interface NavItem {
+  label: string;
+  check?: FeatureCheck;
```

Comment:
same comment about reusing the feature check signature if possible

Reply:
Fixed in [a9c0f1a](https://github.com/getsentry/sentry/pull/77665/commits/a9c0f1aac5788b60053556463c6669a4bbcf4fe7)